persistence/model/src/test/java/gov/va/vro/persistence/repository/ClaimRepositoryTest.java

```diff
@@ -47,6 +47,7 @@ void test() {
     contention1.addAssessmentResult(assessmentResult);
     contention1.addEvidenceSummaryDocument(evidenceSummaryDocument1);
     contention1.addEvidenceSummaryDocument(evidenceSummaryDocument2);
+    contention1.addEvidenceSummaryDocument(count1, evidenceSummaryDocument1.getDocumentName());
```

Comment:
I'm confused slightly here. EvidenceSummaryDocument 1 is added above and you're adding another copy with the same name?

This may be right, but what's going on in this test?

Reply:
removing this, it is unnecessary.